### PR TITLE
#4: UI and General Improvements

### PR DESCRIPTION
**Pull request by 1withspaghetti**
A whole lot of very random changes and a version bump to 1.2.0

Partial list here: https://tattered-dumpling-146.notion.site/Scouting-App-110fc670758b47d5ac87bc299d54119a

### CODE REVIEW

File: src/pages/analytics/AnalyticsTeamPage.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ const AnalyticsPage = () => {
                     <CardContent>
                         <h2 className="text-xl font-bold">Pre Match:</h2>
                         <div className="pl-4 mt-4">
-                            <div>Human Player Location:</div>
+                            <div>{team !== "8248" ? "Human Player Location:" : "Soren's Location:"}</div>
```
Comment (pixelmonaskarion) [inline src/pages/analytics/AnalyticsTeamPage.tsx:154]: "Human Player" is changed to Soren here but not anywhere else on this page (Human Player Scored, etc.)

File: src/pages/scout/ScoutLayout.tsx [hunk 2]
```diff
@@ -109,6 +110,9 @@ const ScoutPage = () => {
             :
                 <NoMatchAvailable />
             }
+            {currentMatchContext?.showConfetti && 
+                <ConfettiWrapper />
```
Comment (pixelmonaskarion) [inline src/pages/scout/ScoutLayout.tsx:114]: I love this

**Review by pixelmonaskarion — APPROVED** [review]
Good changes aside from the small things in my comments.
I'm sad about the PWA being removed but it needs to be.

File: src/pages/scout/ScoutLayout.tsx (hunk 2 above)
Comment (1withspaghetti) [reply src/pages/scout/ScoutLayout.tsx:114]: :D

**Pull request merged**